### DISCUSSION TITLE
I would love NixOS support.

### DISCUSSION THREAD

**Discussion by tompflynn** [NeshHari/XMonad#8] [General]
I installed this on an arch vm using the script and absolutely love it. I know you have listed NixOS support as one of the future goals for the project, but I just wanted to let you know that it's definitely in demand. 

currently I'm stealing bits and pieces from this repo for my own config files on NixOS, but not being able to quickly install the entire environment is a little disappointing. Regardless, I'm not complaining. I had no idea how deep ricing could get before I saw this lol.

**Comment by NeshHari** [comment]
NixOS is the end-goal but I'm still in the midst of learning it whenever I find the time. This "passion project" was created right after my uni finals and now as a full-time software engineer I can barely find the time :cry:. I'll get it done eventually but I'll probably end up using the vertical and optimised eww bar (in the vertical branch - the readme is not updated but the dots are) and completely abandon polybar since eww is just much more configurable imo (slide to reveal, hover, vertical workspaces, etc.), and so that non-xmonad users are also able to use the vertical modules with my eww scripts. Regardless, I do apologise as I am unable to give a timeline for completion :disappointed:.
